zeek_log_transport.sh

```diff
@@ -17,6 +17,48 @@ export PATH="/sbin:/usr/sbin:$PATH"		#Note that cron does _NOT_ include /sbin in
 
 default_user_on_aihunter='dataimport'
 
+get_send_logs() {
+        ##These names were found at https://docs.zeek.org/en/master/script-reference/log-files.html
+        declare -a zeek_logs=("conn" "dce_rpc" "dhcp" "dnp3" "dns" "ftp" "http" "irc" "kerberos" "modbus" "modbus_register_change" "mysql" "ntlm" "ntp" "radius" "rdp" "rfb" "sip" "smb_cmd" "smb_files" "smb_mapping" "smtp" "snmp" "socks" "ssh" "ssl" "syslog" "tunnel" "files" "ocsp" "pe" "x509" "netcontrol" "netcontrol_drop" "netcontrol_shunt" "netcontrol_catch_release" "openflow" "intel" "notice" "notice_alarm" "signatures" "traceroute" "known_certs" "known_hosts" "known_modbus" "known_services" "software" "barnyard2" "dpd" "unified2" "unknown_protocols" "weird" "weird_stats" "broker" "capture_loss" "cluster" "config" "loaded_scripts" "packet_filter" "print" "prof" "reporter" "stats" "stderr" "stdout"  "conn-summary")
```

Comment:
Can you reformat these to be one per line and sorted alphabetically? That'll be easier to tell at a glance if we're missing a file or not.